src/main/java/io/stargate/sgv2/jsonapi/service/schema/collections/CollectionSchemaObject.java

```diff
@@ -48,6 +48,18 @@ public final class CollectionSchemaObject extends TableBasedSchemaObject {
           CollectionLexicalConfig.configForDisabled(),
           CollectionRerankDef.DISABLED);
 
+  // Collection Schema to use for testing: most settings missing, but $lexical
```

Comment:
-1 can we move this to tests ? the MISSING one above was a bad idea that was hard to remove but I would like to.

Reply:
I'll first move (in this PR) it into `DocumentShredder`, if possible. And then figure out further action.